{
	"id": "ELSA-2021-4201",
	"advisories": [
		{
			"content": {
				"id": "ELSA-2021-4201",
				"title": "ELSA-2021-4201:  babel security and bug fix update (MODERATE)",
				"description": "[2.5.1-7]\n- Include the /usr/bin/pybabel binary that runs on Python 3.6 in the\n  python3-babel package\nResolves: rhbz#1967173\n\n[2.5.1-6]\n- Fix CVE-2021-20095\nResolves: rhbz#1955615",
				"severity": [
					{
						"type": "vendor",
						"source": "linux.oracle.com/security",
						"vendor": "MODERATE"
					}
				],
				"references": [
					{
						"source": "linux.oracle.com/security",
						"url": "https://linux.oracle.com/cve/CVE-2021-20095.html"
					},
					{
						"source": "linux.oracle.com/security",
						"url": "https://linux.oracle.com/cve/CVE-2021-42771.html"
					},
					{
						"source": "linux.oracle.com/security",
						"url": "https://linux.oracle.com/errata/ELSA-2021-4201.html"
					}
				],
				"published": "2021-11-16T00:00:00Z"
			},
			"ecosystems": [
				"oracle:8"
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2021-20095",
				"references": [
					{
						"source": "linux.oracle.com/security",
						"url": "https://linux.oracle.com/cve/CVE-2021-20095.html"
					}
				]
			},
			"ecosystems": [
				"oracle:8"
			]
		},
		{
			"content": {
				"id": "CVE-2021-42771",
				"references": [
					{
						"source": "linux.oracle.com/security",
						"url": "https://linux.oracle.com/cve/CVE-2021-42771.html"
					}
				]
			},
			"ecosystems": [
				"oracle:8"
			]
		}
	],
	"detection": [
		{
			"ecosystem": "oracle:8",
			"criteria": {
				"operator": "OR",
				"criterions": [
					{
						"vulnerable": true,
						"package": {
							"name": "python3-babel",
							"architectures": [
								"aarch64",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:2.5.1-7.el8"
								}
							],
							"fixed": [
								"0:2.5.1-7.el8"
							]
						}
					}
				]
			}
		}
	],
	"data_source": "oracle"
}
